docs/_ext/cpp_api.py

```
@@ -296,7 +296,7 @@ def build_doxygen(app: Sphinx) -> None:
     try:
         with urlopen(app.config.qdmi_api_tagfile_url, timeout=30) as response:  # ruff:ignore[suspicious-url-open-usage]
             tagfile.write_bytes(response.read())
-        subprocess.run([doxygen, "Doxyfile"], cwd=app.srcdir, check=True)  # ruff:ignore[subprocess-without-shell-equals-true]
+        subprocess.run([doxygen, "Doxyfile"], cwd=app.srcdir, check=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)